### ISSUE TITLE
When I open up endless sky the frames freeze.

### ISSUE THREAD

**Issue by Aeonis-Myles** [endless-sky/endless-sky#5115]
**Describe the bug**
the game will not open properly and it stays frozen on frames. no matter how long I wait, no matter how many times I uninstall and reinstall the game, It keeps happening. I've tried both downloading it from steam and from github, but neither work properly. all the sounds load properly, and it seems to know what frame it _should_ be on, but it won't work as intended.

**To Reproduce**
1. open up the game
2. it stays on the first frame.
3. maximize, restore down or resize the window, and each time this is done, it refreshes to the frame it should be on at that moment, before immediately freezing again.

**Expected behavior**
I expected the game to open up, do its loading thing, and continue from where i last left off.

**Screenshots**
I can't really screenshot because the problem is with the frames freezing.

**System (please complete the following information):**
 - OS: Windows 8.1
 - Game Source: Steam and git hub, I've tried both and they do the same thing.
 - Version: 9.12

**Link to save file**
steam version: Deleted
Github version: N/A

**Additional context**
I really want to play, and I was planning something really cool for an endless sky video, but I keep running into this error right off the bat, and apparently no one else seems to have encountered this problem, or of they have I haven't seen anyone else report it.

**Comment by Amazinite** [comment]
> Link to save file
> steam version: C:\Program Files (x86)\Steam\steamapps\common\endless-sky
> Github version: C:\Users\ace\Desktop\endless-sky-win64-0.9.12\endless-sky-win64-0.9.12

Um, we don't have access to your computer. Also, these aren't even save file paths. These are the paths to the game executable.

**Comment by Aeonis-Myles** [comment]
Yeah,I'm new to this and completely misunderstood. I deleted all my save files and plugins. all prior versions have worked for me, but this most recent release just doesn't work for some reason.

**Comment by MCOfficer** [comment]
No matter, i would be *very* surprised if this was in any way related to savefiles.

Can you give this a try? https://github.com/endless-sky/endless-sky/issues/4845#issuecomment-615960255

Also, this sounds very similar to [this reddit thread](https://www.reddit.com/r/endlesssky/comments/gsmebw/pplease_help_endless_sky_wont_move_unless_i_move/?utm_source=ifttt)

**Comment by Aeonis-Myles** [comment]
Hi! It's not the same, as I can _technically_ play the game, except in order to do anything i need to play around with the window for anything to move.

**Comment by Amazinite** [comment]
Can you download v0.9.10 from the releases page and see if that works?

**Comment by Aeonis-Myles** [comment]
Uhhhhh. How do I get there?

**Comment by Amazinite** [comment]
https://github.com/endless-sky/endless-sky/releases

**Comment by Aeonis-Myles** [comment]
I already posted this on reddit but I feel that I should post it here too.

Every thing works now. I don't know why, I can't explain it. 0.9.12 works, all versions work now. I did a lot of praying, and I had tried everything I could possibly think of. Today I opened my computer and heard a still, soft voice speaking within me, telling me to turn off the high contrast mode on my computer. I couldn't think of any reason why it would change anything, but I did it anyway. I then opened endless sky and it booted up normally. It had a tiny bit of lag at the beginning, but the lag quickly stopped and finished loading. I can't say it's anything short of a miracle.

I started a new game and just payed off my mortgage, and it works beautifully. I also noticed new missions and dialogue that weren't there before.
